.github/workflows/continous-integration.yml

```diff
@@ -337,7 +337,10 @@ jobs:
         DD_ARGS: --ddtrace --ddtrace-patch-all
       run: |
         make ${{ matrix.test }}
-        poetry run coverage xml -o ${{ github.workspace }}/tests/${{ matrix.test }}-coverage.xml --include="rasa/*"
+        if [[ "${{ matrix.os }}" != "windows-2019" ]]; then
+          mv .coverage ${{ github.workspace }}/${{ matrix.test }}-coverage
```

Comment:
why not keep the `poetry run coverage ...` command?

Reply:
On Windows, there was a problem with where `github.workspace` is on a Windows OS file system. Therefore, `mv` failed when using `shell`, I am not sure if it ever worked. 

We run the tests multiple times anyways and I suspect this moving just overwrites the test jobs finishing early (e.g. ubuntu-latest on Py3.8) with the test jobs that finish late (e.g. Windows on Py3.7). The strategy matrix shows all the combinations we run. 

The suggest implementation (`if os!=windows`) therefore collects only the linux coverage.

Which coverage would we like to collect here ideally?

Reply:
since the issue you described only affects windows and not linux, and you're now running the command only on linux, why not keep the `poetry` command?

The `coverage xml` command generates an XML report, whereas the `mv` command moves internal coverage information to another location, I think this would break the push of the coverage report 🤔

Reply:
Ohh sorry, I understand your question now (I think 🙈).

https://github.com/RasaHQ/rasa/pull/9913/files changed the behavior and uses `mv` instead `poetry run coverage xml`. The `coverage xml` happens in a later step. 

The current was inconsistent:
- single-process used `poetry run coverage xml` https://github.com/RasaHQ/rasa/blob/main/.github/workflows/continous-integration.yml#L340
- multi-process used `mv`
https://github.com/RasaHQ/rasa/blob/main/.github/workflows/continous-integration.yml#L352

Reply:
ok that works 👍🏻 then the `Store coverage reports` report step should not execute on windows, right?

Reply:
Correct, this is captured by the `if` condition of the `Store coverage reports`, so this should be good as it is now

Reply:
oh gosh, I can't read today. Thanks for bearing with me!